helm/hpcc/templates/thor.yaml

```diff
@@ -73,22 +72,21 @@ spec:
       labels:
         run: {{ $thorAgentName | quote }}
         accessDali: "yes"
-        accessEsp: {{ .useChildProcesses | default false | ternary "yes" "no" | quote }}
+        accessEsp: "no"
```

Comment:
NB: this change gets rid of useChildProcesses support for Thor (still supported for eclagent component), because it didn't fully work and complicates the helm chart. Plus for speedy throughput, it is better to persist the Thor instance and allow it to run multiple jobs (via multiJobLinger).